reconcile/gitlab_members.py

```diff
@@ -0,0 +1,144 @@
+import sys
+import logging
+
+import utils.gql as gql
+from utils.config import get_config
+from utils.gitlab_api import GitLabApi
+
+GROUPS_QUERY = """
+{
+    instances: gitlabinstance_v1{
+        managedGroups
+    }
+}
+"""
+
+USERS_QUERY = """
+{
+    users: users_v1{
+        redhat_username
+        roles{
+            permissions{
+                ... on PermissionGitlabInstance_v1{
+                    name
+                    group
+                }
+            }
+        }
+    }
+}
+  """
+
+BOTS_QUERY = """
```

Comment:
now that we have both users and bots, would it make more sense to have a single query?
example: https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/jenkins_roles.py#L26

Reply:
and also if you can make the queries look more like that example visually?

Reply:
Does doing a single query change anything other than style? Or do you want me to filter by roles instead of by users and bots separately like in the example.

Reply:
1 query is usually faster then 2, depending on the payload of course. in this case, it probably doesn't make much difference. so i would say that this is more about style, and maybe a bit about avoiding code duplication. what is more likely to change in the future wrt queries is the query for these specific roles. we are less likely to need anything other then redhat_username. these were my thoughts. i'm leaving this for you to decide, either way is fine :)